mix.exs

```diff
@@ -35,7 +28,6 @@ defmodule GraphqlMarkdown.MixProject do
   defp deps do
     [
       {:credo, "~> 1.5", only: [:dev, :test], runtime: false},
-      {:excoveralls, "~> 0.14.2", only: :test},
```

Comment:
Where this is just a test dependency, do we really need to increment the version?

Reply:
removing this one as it s really is useless now